newdle/client/package.json

```diff
@@ -5,13 +5,19 @@
   "dependencies": {
     "flask-urls": "^0.1.0",
     "indico-sui-theme": "0.0.3",
+    "prop-types": "^15.7.2",
     "react": "^16.8.6",
     "react-dom": "^16.8.6",
-    "react-scripts": "3.0.1"
+    "react-gravatar": "^2.6.3",
+    "react-scripts": "3.0.1",
+    "semantic-ui-react": "^0.87.3"
   },
   "devDependencies": {
     "flask-urls.macro": "0.0.1",
-    "node-sass": "^4.12.0"
+    "node-sass": "^4.12.0",
+    "chai": "^4.2.0",
```

Comment:
why do we need to explicitly depend on it? i can't see it referenced anywhere in our code. is it a peer dependency?

Reply:
There is one reference. `UserMenu.test.js`